src/test/kotlin/in/rcard/assertj/arrowcore/EitherAssert_hasRightValueSatisfying_Test.kt

```diff
@@ -0,0 +1,42 @@
+package `in`.rcard.assertj.arrowcore
+
+import arrow.core.Either
+import `in`.rcard.assertj.arrowcore.errors.EitherShouldBeRight.Companion.shouldBeRight
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.assertj.core.util.FailureMessages.actualIsNull
+import org.junit.jupiter.api.Test
+
+
+internal class EitherAssert_hasRightValueSatisfying_Test {
+
+    @Test
+    internal fun `should fail when either is null`() {
+        val actual: Either<Int, String>? = null
+        assertThatThrownBy { EitherAssert(actual).hasRightValueSatisfying { } }
```

Comment:
You still use the constructor instead of the factory method in all these tests. Can you change it?

Reply:
Sorry, we missed it 😅